src/UI/citizens/components/select-date-time/StyleCitizensSelectDateTime.js

```diff
@@ -0,0 +1,277 @@
+import styled from 'styled-components';
+
+const Container = styled.div`
+  position: relative;
+  width: 1024px;
+  height: 488px;
```

Comment:
Height se nikada ne postavlja explicitno, to je rezultat sadržaja tog kontejnera. Postavi min-height